spec/src/main/asciidoc/ch03-entity-operations.adoc

```diff
@@ -2023,13 +2021,14 @@ public class Employee {
     @Basic
     protected String employeeNumber;
 
-    @OneToMany()
+    @OneToMany
     protected List<Dependents> dependents;
 
-    @OneToMany()
+    @OneToMany
+    @NamedEntityGraphAttributeNode
```

Comment:
I think it might be nicer if we added additional examples instead of altering the existing ones. Maybe also add an example for the `@NamedEntitySubgraph`?

Reply:
Oh I lie, I did change them. Did it because I was originally planning on deprecating the old way. I can revert that.